src/components/elements/Tabs/Default/TabButton.tsx

```diff
@@ -34,21 +34,22 @@ export const TabButton = ({
       className={classNames(
         className,
         `w-full items-center focus:outline-none disabled:text-neutral-900`,
-        "border",
+        "",
         item.done && "peer",
-        lastItem || selected ? "border-b" : "border-b-0",
+        lastItem || selected ? "" : "",
         selected
-          ? "border-neutral-100 border-r-white bg-white text-neutral-1000 peer-aria-checked:shadow-t-secondary"
+          ? `border-l-4 border-l-[#27A9E0]
+          bg-white text-neutral-1000`
```

Comment:
This is a little confusing broken onto two lines, please join back up.

Reply:
done